apps/reference/docs/guides/cli/custom-hostnames.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: custom-hostnames
+title: Custom Hostnames
+description: Configuring a Custom Hostname for your Supabase project.
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+:::note
+
+Custom Hostnames are currently in beta, and are being slowly made available to projects.
+
+:::
+
+:::note
+
+The commands in this guide require Supabase CLI 1.10.1+
+
+:::
+
+## Overview
+
+Custom hostnames allow you to present a branded experience to your users.
+
+Before you start, here are some prerequisites:
+
+- [Install the Supabase CLI](/docs/guides/cli)
+- Ensure that you've configured the CLI for your account
+- Create a Supabase project
+- Ensure that you've subscribed to the Custom Hostnames add-on
+
+## Configuring a Custom Hostname
```

Comment:
Suggest getting rid of this heading and adding h2 headings for each of the steps so that it appears in the table of contents on the right to allow users to skim through the steps easily. Something like:
1. Set up a CNAME record
2. Create the custom hostname
3. Set up TXT records
4. Serve traffic on new hostname